bioimageio/core/block_meta.py

```diff
@@ -161,8 +161,8 @@ def local_slice(self) -> PerAxis[SliceInfo]:
         return Frozen(
             {
                 a: SliceInfo(
-                    self.padding[a].left,
-                    self.padding[a].left + self.inner_shape[a],
+                    self.halo[a].left,
+                    self.halo[a].left + self.inner_shape[a],
                 )
```

Comment:
this is the actual fix!! I'm glad you found! Maybe we make a sparate PR for it if we need more time incorporating your new verbose features?